packages/runtime/container-runtime/src/garbageCollection.ts

```diff
@@ -1168,6 +1177,26 @@ export class GarbageCollector implements IGarbageCollector {
         if (nodeStateTracker && nodeStateTracker.state !== UnreferencedState.Active) {
             this.inactiveNodeUsed("Revived", toNodePath, nodeStateTracker, fromNodePath);
         }
+
+        if (this.tombstones.includes(toNodePath)) {
+            const nodeType = this.runtime.getNodeType(toNodePath)
+
+            let eventName = "GC_Tombstone_SubDatastore_Revived";
+            if (nodeType === GCNodeType.DataStore) {
+                eventName = "GC_Tombstone_Datastore_Revived";
+            } else if (nodeType === GCNodeType.Blob) {
+                eventName = "GC_Tombstone_Blob_Revived";
+            }
+
+            this.mc.logger.sendTelemetryEvent({
+                eventName,
+                isSummarizerClient: this.isSummarizerClient,
+                pkg: packagePathToTelemetryProperty(this.getNodePackagePath(toNodePath)),
```

Comment:
I don't see the point of doing this because the package path will most likely be undefined. If a node is tombstoned, it hasn't been loaded in a while and so the package path will be undefined. Can you add tests to validate this scenario?
Also, if the packagePath is undefined, this will result in an assert. See my other comment in dataStores.ts.

Reply:
Ok will do